src/data-workspace/entry-form.js

```diff
@@ -72,12 +62,28 @@ export const EntryForm = React.memo(function EntryForm({ dataSet }) {
                     formType={formType}
                 />
             )}
-
+            <EntryFormErrorSpy />
             <Component dataSet={dataSet} globalFilterText={globalFilterText} />
         </>
     )
 })
 
+/*  Used to sync store with errors from form
+    In its own component to prevent unecessarily re-renders in the tree */
```

Comment:
```suggestion
/**
 * Used to sync global store with errors from final-form.
 * Because of the `errors` subscription in `useFormState`, this re-renders
 * on every form input change if there is an error in the form. Therefore, this 
 * has its own component to prevent unnecessary re-renders below it in the tree
 */
```